### PR TITLE
Actualize action sheet

### PR DESCRIPTION
**Pull request by SAnatoliiS**

### CODE REVIEW

File: Example/src/screens/Menus.tsx [hunk 1]
```diff
@@ -13,18 +13,119 @@ import {
...
 import { ExampleSection } from '../components/ExampleSection';
 import { ExampleScreen } from '../components/ExampleScreen';
 
+const onPress = () => {
+    console.log('Action');
+};
+
+const foregroundList: UIActionSheetContainerChildType[] = [
+    <UIPopup.ActionSheet.CustomAction key="1">
+        <UIPopup.ActionSheet.PrimaryColumn>
```
Comment (savelichalex) [inline Example/src/screens/Menus.tsx:37]: Can we somehow accept single action and wrap it in PrimaryColumn if not other options aren't provided? 🤔 Looks too verbose for now

File: kit/popups/src/ActionSheet/UIActionSheetAction.tsx [hunk 2]
```diff
@@ -1,49 +1,52 @@
...
     onPress,
 }: UIActionSheetActionProps) => {
-    const variant: UIBoxButtonVariant = getActionVariant(type);
-    const boxButtonType: UIBoxButtonType = getBoxButtonType(type);
-    return (
-        <View key={title}>
-            <UIBoxButton
+    if (type === UIActionSheetActionType.Cancel) {
+        return (
+            <TouchableWrapper
```
Comment (savelichalex) [inline kit/popups/src/ActionSheet/UIActionSheetAction.tsx]: Why it isn't done with Foreground?

**Review by savelichalex — COMMENTED** [review]
It's hard to review for now as it contains commits from another PR. Let's change base or wait for another PR to be merged.

File: Example/src/screens/Menus.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply Example/src/screens/Menus.tsx:37]: Yes, we can. We don't need PrimaryColumn for this purpose. https://github.com/tonlabs/UIKit/pull/1461/files#diff-cfed5573f8056947bb7e2760d99ca09c630ecea6b3df46283987bb8711305632R369-R373

File: kit/popups/src/ActionSheet/UIActionSheetAction.tsx (hunk 2 above)
Comment (SAnatoliiS) [reply kit/popups/src/ActionSheet/UIActionSheetAction.tsx:16]: It is done

Comment (SAnatoliiS) [reply kit/popups/src/ActionSheet/UIActionSheetAction.tsx]: Because this button differs from the UIForeground in font and color. I should add it.

Comment (SAnatoliiS) [reply kit/popups/src/ActionSheet/UIActionSheetAction.tsx]: Done

File: Example/src/screens/Menus.tsx (hunk 1 above)
Comment (savelichalex) [reply Example/src/screens/Menus.tsx:37]: Either link is broken or I just doesn't understand explanation :)

**Review by savelichalex — APPROVED** [review]

File: Example/src/screens/Menus.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply Example/src/screens/Menus.tsx:37]: The link was broken :) You can see example of using simple action in bottom of this file (`UIPopup.ActionSheet.Action`)

Comment (savelichalex) [reply Example/src/screens/Menus.tsx:37]: Got it :) But actually I meant the Foreground api. Like that next one would be valid:
```ts
<UIPopup.ActionSheet.CustomAction>
  <UIPopup.ActionSheet.Action title="Press me" />
</UIPopup.ActionSheet.CustomAction>
```

File: kit/popups/src/UIForeground/hooks.ts [hunk 3]
```diff
@@ -41,3 +41,49 @@ export function useMergedColumnStatus(
...
+        return true;
+    };
+}
+
+export function useCheckChildren(
+    children: React.ReactNode,
+    allowedChildrenComponents: Record<string, unknown>,
+    errorMessage: string,
+): boolean {
+    return React.useMemo(() => {
```
Comment (savelichalex) [inline kit/popups/src/UIForeground/hooks.ts:82]: Let's maybe disable it in `production` mode?

**Review by sertony — APPROVED** [review]

**Pull request merged**